### ISSUE TITLE
Cannot change node key

### ISSUE THREAD

**Issue by PawelPodkalicki** [adshares/ads#57]
I added private key to `key/key.txt` file.
Called `change_node_key` with matching public key.
Result `Node key changed` was returned.
Node stopped immediately with error.

```
[1528275309] ERROR: bad current node key for 0001 in 0001:00000005
[1528275309] ERROR, have invalid message 0001:00000005 !!!
[1528275309] BAD message 0001:00000005 00010000000503C6 EAC6A0B6737D8DEF98E19E668AD1576C891F871579AEE090970C43E76160CD0B
[1528275309] BAD insert blk/5B1/7A160/03_0001_00000005.und to blk/5B1/7A160/03_0001_00000005_EAC6A0B6737D8DEF98E19E668AD1576C891F871579AEE090970C43E76160CD0B.und failed
[1528275309] BAD insert blk/5B1/7A160/03_0001_00000005.msg to blk/5B1/7A160/03_0001_00000005_EAC6A0B6737D8DEF98E19E668AD1576C891F871579AEE090970C43E76160CD0B.msg succeeded
```

Full log is available **20180606_1058_logs_node_key_chg_stop**

**Comment by PawelPodkalicki** [comment]
Error exists no matter, if in `key.txt` are two keys (old and new), or single new key.

**Comment by PawelPodkalicki** [comment]
In b7c3e67a321fb9fe0b3419a34cd0ae2d80aba110 node doesn't stop. For change_node_key response is 
```
{
    "error": "read: Connection reset by peer"
}
```
Error occurred in 3 of 4 attempts. Full log is available in **20180618_1144_logs_chg_nk_conn_rst.zip**

**Comment by PawelPodkalicki** [comment]
Fixed in 728a076f167db31e15681ca950b7ac1dd8ccd246
